opengrok-indexer/src/main/java/org/opengrok/indexer/index/IndexDatabase.java

```diff
@@ -353,27 +354,38 @@ private void markProjectIndexed(Project project) {
         // Successfully indexed the project. The message is sent even if
         // the project's isIndexed() is true because it triggers RepositoryInfo
         // refresh.
-        if (project != null) {
-            // Also need to store the correct value in configuration
-            // when indexer writes it to a file.
-            project.setIndexed(true);
-
-            if (env.getConfigURI() != null) {
-                Response r = ClientBuilder.newClient()
-                        .target(env.getConfigURI())
-                        .path("api")
-                        .path("v1")
-                        .path("projects")
-                        .path(Util.URIEncode(project.getName()))
-                        .path("indexed")
-                        .request()
-                        .put(Entity.text(""));
-
-                if (r.getStatusInfo().getFamily() != Response.Status.Family.SUCCESSFUL) {
-                    LOGGER.log(Level.WARNING, "Couldn''t notify the webapp that project {0} was indexed: {1}",
-                            new Object[] {project, r});
-                }
-            }
+        if (project == null) {
+            return;
+        }
+
+        // Also need to store the correct value in configuration
+        // when indexer writes it to a file.
+        project.setIndexed(true);
+
+        if (env.getConfigURI() == null) {
+            return;
+        }
+
+        Response r;
+        try {
+            r = ClientBuilder.newClient()
+                    .target(env.getConfigURI())
+                    .path("api")
+                    .path("v1")
+                    .path("projects")
+                    .path(Util.URIEncode(project.getName()))
+                    .path("indexed")
+                    .request()
+                    .put(Entity.text(""));
+        } catch (RuntimeException e) {
+            LOGGER.log(Level.WARNING, "Couldn''t notify the webapp that project {0} was indexed: {1}",
```

Comment:
Not sure the ramification, but usually we would see used the `log(Level, String, Throwable)` method when an entire `Throwable` is reported (versus e.g. just the `getMessage()`)

Reply:
True, changed.